src/index.js

```diff
@@ -72,6 +72,7 @@ export class Circuit extends HeadlessCircuit {
         return this.makePaper(elem, this._graph);
     }
     makePaper(elem, graph, opts) {
+        var circuit = this;
```

Comment:
This should be a const, as it's not supposed to be changed.
```suggestion
        const circuit = this;
```